src/current/v24.3/data-resilience.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Data Resilience
+summary: An overview of how CockroachDB deployments can achieve resilience with built-in features and tooling.
+toc: true
+---
+
+CockroachDB provides built-in [**high availability (HA)**](#high-availability) features and [**disaster recovery (DR)**](#disaster-recovery) tooling to achieve operational resiliency in various deployment [topologies]({% link {{ page.version.version }}/topology-patterns.md %}) and use cases.
+
+- HA features ensure continuous access to data without interruption even in the presence of [failures]({% link {{ page.version.version }}/demo-cockroachdb-resilience.md %}) or disruptions to maximize uptime.
+- DR tools allow for recovery from major incidents to minimize downtime and data loss.
+
+<image src="{{ 'images/v24.3/resilience.svg' | relative_url }}" alt="Diagram showing how HA features and DR tools create a resilient CockroachDB deployment." style="width:80%" />
+
+You can balance required SLAs and [recovery objectives]({% link {{ page.version.version }}/disaster-recovery-overview.md %}) with the cost and management of each of these features to build a resilient deployment.
+
+{{site.data.alerts.callout_success}}
+For a practical guide on how CockroachDB uses Raft to replicate, distribute, and rebalance data, refer to the [CockroachDB Resilience demo]({% link {{ page.version.version }}/demo-cockroachdb-resilience.md %}) demo.
+{{site.data.alerts.end}}
+
+## High availability
+
+- [**Raft replication**]({% link {{ page.version.version }}/architecture/replication-layer.md %}): CockroachDB's built-in Raft replication stores data safely and consistently on multiple nodes to ensure no downtime even during a temporary node outage. [Replication controls]({% link {{ page.version.version }}/configure-replication-zones.md %}) allow you to configure the number and location of [replicas]({% link {{ page.version.version }}/architecture/glossary.md %}#replica) to suit a deployment.
```

Comment:
Was it intentional to call this by the underlying tooling instead of the outcome? We've had success referring to this as "multi-active availability"

Reply:
I have switched to multi-active availability. It was intentional on my first draft to just talk about tooling; however, we do actually define multi-active and have a whole page dedicated to this in the docs, so I have switch to multi-active here as well.